lib/facter/facts/freebsd/zfs_featurenumbers.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Facts
+  module Freebsd
+    class ZfsFeaturenumbers
+      FACT_NAME = 'zfs_featurenumbers'
+
+      def call_the_resolver
+        require 'facter/resolvers/solaris/zfs_resolver'
```

Comment:
Could you please delete this require and move the zfs and zpool resolvers to /lib/facter/resolvers/zfs.rb and /lib/facter/resolvers/zpool.rb. The resolvers in lib/facter/resolvers are loaded for all platforms. The resolvers which reside in different folders inside  /lib/facter/resolvers are loaded only for specific platforms.